### PR TITLE
Added header social logo and welcome text

### PR DESCRIPTION
**Pull request by alexmedinasf**
Index:
Added the main html code with:

Header
Social Media logos with working links to redirect
Greetings with name and a Who Am I text
Css:

Edited letters font
Edited background and header colors
Edited Font colors
Edited background image size and placement
Edited text size and placement
Edited Social media logos size and placement

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+	<header class="menu-bar">
+        <a href="#" class="name">Alexandre Ferreira</a>
+        <a href="#" class="menu"><img src="./images/menu.png" alt="menu"></a>
+    </header>
+	<section class="intro">
+        <h1>Hello! My name is Alexandre</h1>
+        <h2>I'm a Microverse Student</h2>
```
Comment (bobb-Rob) [inline index.html:20]: - [ ] Please I suggest you stick to the design as much as possible including the text. Kindly replace I**_'m a Microverse Student_** with the text that is given in the Figma design.

File: README.md [hunk 2]
```diff
@@ -1,48 +1,26 @@
...
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
```
Comment (bobb-Rob) [inline README.md:26]: - [ ] Please ensure that your README file is very detailed showing all the relevant heading as shown in the microverse [Readme template](https://github.com/microverseinc/readme-template). Also, I notice you use the file template of a database project as you currently have unrelated files such as `schema.sql`, `queries.sql`, and `data.sql` in your project. Please remove all unrelated files.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
### Hi @alexmedinasf , 

Good job so far! 👏👏👏
You did very well implementing almost all the requirements of this milestone. 🥳 
But there are some issues that you still need to work on to go to the next project but you are almost there!  😃 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	<title>Alexandre Portfolio</title>
+	<link rel="stylesheet" href="./css/style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin> 
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+	<nav class="menu-bar">
+        <button type="button" class="name"> Alexandre Ferreira </button>
+        <button type="button" class="menu"> <img src="./images/menu.png" alt="menu"> </button>
```
Comment (leonardodiasb) [inline index.html:16]: - [ ] You did a good job changing the links to `<button>` tag, but it seems that you forgot to re-style the buttons. Buttons have a different default style and in most cases, they should be styled. Kindly change the style to match the Figma template and make sure the hamburger button is on the right side.
 - Your design: 
 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @alexmedinasf, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No Linter error ✔️
 - Correct use of GitHub flow ✔️
 - Very descriptive Pull Request(PR) ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (alexmedinasf) [reply index.html:16]: Hey @leonardodiasb , thank you for the review. I changed the buttons style and corrected the errors, do I need to make a new pull request?

Comment (leonardodiasb) [reply index.html:16]: Hi @alexmedinasf, no you don't. Just submit this PR again.

**Review by cwaku — APPROVED** [review]
Hi @alexmedinasf 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
